bindata/network/multus/multus.yaml

```diff
@@ -0,0 +1,58 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the Multus networking component on each node.
+spec:
+  selector:
+    matchLabels:
+      app: multus
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: multus
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      serviceAccountName: multus
+      containers:
+      - name: kube-multus
+        image: nfvpe/multus:latest
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+          limits:
+            cpu: "100m"
+            memory: "50Mi"
+        env:
+        - name: MULTUS_CONF_FILE
```

Comment:
Dan this might get clobbered by the entrypoint script. What I have been testing with to enable the auto functionality is to add args to the entrypoint, with:

```
        command: ["/entrypoint.sh"]
        args: ["--multus-conf-file=auto"]
```

Reply:
@dougbtv yeah, nfvpe/multus needs a rebuild to pick up the "auto" functionality; it doesn't have it currently.